packages/webcomponents/package-lock.json

```diff
@@ -263,148 +263,6 @@
 			"integrity": "sha512-u5J+m2GSGGGQDKzIBNjaHPLEHJIYDA05kyzWHBxq0qcZfbbVtMVH8x+P2B36YYyYOBWQIIsJjYmNJ05mbmqeWA==",
 			"dev": true
 		},
```

Comment:
Does those changes make sense?

Reply:
oh hm, try running `npm run bootstrap` from the root of this repo, it may revert some of this back

also were you able to confirm that the latest version of s.min.js is using es5 with this update?

Reply:
Yes confirmed

Reply:
running bootstrap doesn't seem to affect this file.